CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Head
 
+- Added: SCSS integration test.
```

Comment:
I think we should take this out as it's an internal change, that doesn't affect the semver e.g. "Added" = minor release, rather than patch.